node-notes-cli/notes.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+const jsonData = require('./data.json');
+const info = JSON.stringify(jsonData.notes, null, 2);
+const command = process.argv[2];
+const index = process.argv[3];
+const updateString = process.argv[4];
+
+if (command === 'read') {
+  let newInfo = info.substring(1, info.length - 1);
+  newInfo = newInfo.substr(1, newInfo.length - 1);
+  newInfo = newInfo.replace(/"/g, '');
+  newInfo = newInfo.replace(/,/g, '');
```

Comment:
This isn't quite right, what if the user includes a comma in their input? Try just looping through your notes object with `for ... in`